tritium-metrics/src/main/java/com/palantir/tritium/event/metrics/MetricsInvocationEventHandler.java

```diff
@@ -42,16 +47,55 @@
     private final MetricRegistry metricRegistry;
     private final String serviceName;
 
+    //consider creating annotation handlers as separate objects
```

Comment:
+1, I think we'll want to have mapping of method signature to handler to implement https://github.com/palantir/tritium/issues/14

Reply:
I'm thinking about ways to do this.  Most clean way seems to be:

1) MetricsInvocationEventHandler has a Multimap<MethodSignature, MetricsAnnotationHandler>
2) Each MetricsAnnotationHandler contains a Map<MethodSignature, Annotation>

Sort of begs the question if the MetricsInvocationEventHandler needs awareness of a the Annotation Handlers or whether the annotation handlers can be completely generic invocation handlers.

Reply:
yeah, I think we can leave this as-is for now and extract flesh out the annotation handlers separately